public/documents/fari-rpgs/charge-rpg.md

```diff
@@ -2636,6 +2636,101 @@ Granting progress on the `party sheet` could happen when milestones or long-term
 >
 > `Supporting details` can be upgraded or downgraded based on filling the `progress` and `danger` clocks, respectively. The `danger` clock might be filled as a consequence, while the `progress` clock is filled each milestone and as the result of `long-term projects`. Ticks on the `progress` clock can be burned as momentum on a one-for-one basis.
 
+# Threat Level Extra | Extras
+
+**An alternate way to think adversity in Charge**
+
+\- By othelarian
+
+---
+
+Sometimes, ennemies are so powerful they have a direct effect over characters, hindering their abilityto fight back.
+
+Threat Level is an extra about adversity and how it can affect the character's dice rolls.
+
+## What is a Threat Level?
+
+While adversity doesn't affect directly the character, it's ok to use position and effect. But when it has an impact, it can be less obvious on how to deal with it and make it count mechanically speaking.
```

Comment:
I'm not sure I understand the messages you're trying to convey here (tu peux répondre en français qu'on se comprenne bien)

Reply:
I made a mistake, I was thinking "as long as" instead of "While". I will use the therabidbanana suggestion to rework this part.